communications/newsletters/newsletter05_May2019.md

```diff
@@ -0,0 +1,52 @@
+# The Turing Way -  Book dash reports
+
+Dear friends of the Turing Way,
+
+it's been quiet here for a while as we've been so busy running book dashes that resulted in amazing new contributions to the book.
+
+## GitHub repository update
+After we announced in our last newsletter that we split off the handbook in a separate GitHub repository to the main project repository, we decided to move the book back into the main Turing Way repository. 
+We realized that there wasn't a technical reason for having the book in a separate repository and splitting discussions across two repositories is rather confusing. 
+Thus, you can find everything at https://github.com/alan-turing-institute/the-turing-way and we have now also preserved various releases of the handbook [on zenodo](https://doi.org/10.5281/zenodo.3233853).
+
+## Book dashes 
+
+We were busy organising and running two very succcessful book dashes in May. 
+Our events in Manchester and London were great opportunities to grow the Turing Way community and we were really lucky to be joined by an illustrator from [scriberia](http://www.scriberia.co.uk/) who captured our book dahes wonderfully:
+
+![](https://pbs.twimg.com/media/D6zGuPiW4AI3m8s.jpg)
```

Comment:
Can we add this image to the github repo and link to that?

Reply:
Will do, where do figures and images live now that we moved them around a bit?

Reply:
Aaaah - rubbish. Make a figures folder for the newsletter?